### DISCUSSION TITLE
Problem running VS Code

### DISCUSSION THREAD

**Discussion by asajid03** [microsoft/vscode-discussions#417] [Extension Development QnA] [answered]
HI Everyone.
I am facing this issue for several days and I have almost done everything but still issue persue. I was installing flutter and I do not remember what i did wrong due to which I ended up my terminal like this.
kali@localhost:~% ls
Command 'ls' is available in the following places
 * /bin/ls
 * /usr/bin/ls
The command could not be located because '/usr/bin:/bin' is not included in the PATH environment variable.
ls: command not found
 Now whatever command I am trying to type is vscode it ends up with the same output.
THANKS in Advance.

**Comment by isidorn** [comment, marked as answer]
This should be an issue in the VS Code repo https://github.com/microsoft/vscode/issues
